tests/integration/utils.py

```diff
@@ -626,7 +626,9 @@ def pm_name(pm_change: Path) -> str:
 
 
 def affected_package_managers(pm_changes: tuple[Path, ...]) -> set[str]:
```

Comment:
You would still need "multiple" in `EXTRA_PMS` for it to be properly excluded though. I think adding it back with a comment is the correct thing to do ATM. Everything else seems to work fine.